### PR TITLE
Modified the problem of incomplete block browsing address

### PR DESCRIPTION
**Pull request by shefan2017**
[For Issues #76 ] Modified the problem of incomplete block browsing address

### CODE REVIEW

File: src/core/blocks.js [hunk 1]
```diff
@@ -104,6 +105,7 @@ private.proposeCache = {};
 private.lastPropose = null;
 
 const FULL_BLOCK_QUERY = "SELECT " +
+  "a.address"+
```
Comment (yanyiwu) [inline src/core/blocks.js]: 这个pr测试了吗？看这里是有sql语法错误吧？少了个逗号？

**Comment by yanyiwu** [comment]
travis build failed ，麻烦修复一下。

**Comment by yanyiwu** [comment]
Can you post your asch wallet adress, we will send you some xas reward for your contribution. Thank you again @shefan2017

**Comment by shefan2017** [comment]
A7XBsecVtHEze4Lfrscmg2DEDChETRknZm

**Comment by ByCnck** [comment]
修改不全，加了个字段，序列化后再解析的时候字段会错位。暂时先全部回滚了

```
info 2018-02-06 12:49:50 015 loader.js:370 Blockchain ready 
info 2018-02-06 12:49:50 021 dapps.js:2415 start to launch 0 installed dapps 
warn 2018-02-06 12:49:50 023 dapps.js:2431 Asch witness club dapp is not found 
info 2018-02-06 12:49:50 027 peer.js:417 Peers ready, stored 10 
info 2018-02-06 12:49:50 218 loader.js:187 Check blockchain on 45.76.98.139:80 
----------------------------tx_raw { a_address: '12840465545738645956',
  b_id: '0',
  b_version: '4231680',
  b_timestamp: 64,
  b_height: 17581064340366897000,
  b_previousBlock: '1',
  b_numberOfTransactions: '0',
  b_totalAmount: '10000000',
  b_totalFee: '0',
  b_reward: '2262',
  b_payloadLength: '6827e842100761d4e24ecee41c475089cf8ca636b956d44b43d3a1320f910d5c',
  b_payloadHash: 'bdd79c6ec2aafb66c4495aec8661c16c117da42919465efcaf1306b811bc0c4e',
  b_generatorPublicKey: 'c1f1f5b47b93eeb7df2cd67f8f7ca21baeba6d30367c5f8cdadd9746e7a550a78e3f5afabb27cd775697cd1b9f52482e0383114841372027a6e4f8931410c601',
  b_blockSignature: '15303520775205037928',
  t_id: '3',
  t_type: 4231671,
  t_timestamp: NaN,
  t_senderPublicKey: '12448977009388136020',
  t_senderId: '',
  t_recipientId: '0',
  t_amount: '10000000',
  t_fee: '1ed2043fe03a97992cabae92506fc3c25c68da9c31b114035e60fd99238f29871bfe739d2293e5e3c329958b81a9fb2141350174acab3a9b80a1e12a4e815906',
  t_signature: '',
  t_signSignature: '',
  t_args: '',
  t_message: '',
  s_publicKey: '',
  d_username: '+008abb67fc6e313212ab7550c6552f2e415708084742dafa24ba502310cf11e0,+03d0d46fa37ea40ac6cbb83433e8310d5f40b123e292f84d30d410a0c52da88e,+0665038f9ccd22b6d0055b2555383b637f90bfba5e3e6908b46ec05f6634fc60,+06e8b9a6cbf3ea76120f6abd60435a924127a64a6d3d9af9ef7fed30ddac2737,+0adb4d3da93c1309ab322b31b33fe7fabdc60b2f6219e07817de205b5d2109bc,+0b0d7faf654a1cce643b685cd1dd9c7e6f4e2fcf4e630384410fec7a974634bb,+0d3cb36941be23e929d0bdfeb66c6c3ec475013f8446f735eb2840ea2140fb81,+0e5e68360c01ba31636f19268bf4f6d864ab5de8949ddfac7a294b9c95540d45,+0f454ae85c28295544c2fba3c8531b09bae6cae8435f75d9800037291bd7b62b,+1217d403742cf0e73dc1c73a1e56c402e80acede3a934872865ab0b1bd7c156b,+125a83eb06bb1bdd5adea1fb5e36f67549d750eb7d174f92e13f8c41336563eb,+1816f478d408dd0819a085f174d45f7ada76aaccfeeabba6d45eb991fd9b5dcd,+2095ea8f44eae6cb82dced35de5cde4c3879c5c927fda43071381e3920f9fcff,+21e9da9daf1c9706597c6771f3e918921df4c2f92e68c4c61edc8f7454c38b5a,+231c53d11288e8e69e1b881327681ce5ca9d3583380e2bcba4d13f452d693d61,+23962c958cf0ecd9bafb188e6057a15200d68581b7459b9fd1d8bebbde419b89,+2726f98329f4df9617cfb04e2859bdb96e29c3b377c4517592d0f1b41c4f09fb,+27d04cf6422052ae6a43c8e81a64f010e012c92e35808e71fe9329cae6057166,+2941d9c6b0521a5b10fb7a17d8f8602aee5814632c2b6d66719375ede01f2daf,+29491ecb67d0a1e1bc88d042ab025b3f888d0ea13e66f97a9628ebdebc934e38,+2a98da7f5d3fbcbc01bc0737cba5c7d9a0135f7a6b65ecbfe624beff257d35e1,+2aa47647feb5122fa3151dc1d3d901affd3e6032bf2b9017f051e53d857987f6,+2c0e19f2d072bb730f40860316a3c22d2820bbd39baa9ba4ba262f3370382fb7,+2fe6dabf4ba378ca3f8264c741bf39c6e68a5aa6507d25b5eb4095a890ddbd61,+30c674e09c3b8ff047063244dc4bbdffa66cecb976e702f91807fc72834f6867,+349d04c05a0e035dde502e9ac06228e7193ac5dd4eeaedd912865b4f151ce8b9,+3b406079ed99264253b101b925a5a0c8fcb52fab7b3f2a7d89275f9ad83b525d,+39d04e14a801f0df807d116ed793a0b0843d2cd8d73ec056170aca68b99d1945,+3cc9d06587c0e73c5342fada5e72212cbffa03094dea77ec7268b252089421a4,+489c8b074567c9b1e5e45b8dec2943733ec9a30e01d5778716ac6d7a83111a65,+49226021e851cfc096c65c368e8b8bad9ade434c491c907e9612fe1fdd439fc8,+4ee80e90ad6fbff974e76e9be740cd4cf698c102377ad95cce7e310a24ee65f9,+516cab33f43657d68049038526ed357b08f740da6f222228756940ea38622dd3',
  v_votes: '',
  m_min: 0,
  m_lifetime: 0,
  m_keysgroup: '',
  dapp_name: '',
  dapp_description: '',
  dapp_tags: '',
  dapp_type: 0,
  dapp_link: '',
  dapp_category: 0,
  dapp_icon: '',
  dapp_delegates: '',
  dapp_unlockDelegates: 0,
  it_dappId: '',
  it_currency: '',
  it_amount: '',
  ot_dappId: '',
  ot_outTransactionId: '',
  ot_currency: '',
  ot_amount: '',
  t_requesterPublicKey: '',
  t_signatures: '',
  st_content: '',
  issuers_name: '',
  issuers_desc: '',
  assets_name: '',
  assets_desc: '',
  assets_maximum: '',
  assets_precision: 0,
  assets_strategy: '',
  assets_allowWriteoff: 0,
  assets_allowWhitelist: 0,
  assets_allowBlacklist: 0,
  flags_currency: '',
  flags_flag: 0,
  flags_flagType: 0,
  issues_currency: '',
  issues_amount: '',
  transfers_currency: '',
  transfers_amount: '',
  acls_currency: '',
  acls_flag: 0,
  acls_operator: '',
  acls_list: '' }
----------------------------tx { id: '3',
  height: 17581064340366897000,
  blockId: '0',
  type: 4231671,
  timestamp: NaN,
  senderPublicKey: '12448977009388136020',
  requesterPublicKey: '',
  senderId: '',
  recipientId: '0',
  amount: 10000000,
  fee: 1,
  signature: '',
  signSignature: '',
  signatures: null,
  confirmations: undefined,
  args: null,
  message: '',
  asset: {} }
fatal 2018-02-06 12:49:50 412 app.js:227 uncaughtException { message: 'Unknown transaction type 4231671',
  stack: 'Error: Unknown transaction type 4231671\n    at Transaction.dbRead (/root/github.com/AschPlatform/asch/src/base/transaction.js:738:13)\n    at Object.private.readDbRows (/root/github.com/AschPlatform/asch/src/core/blocks.js:483:52)\n    at /root/github.com/AschPlatform/asch/src/core/blocks.js:1155:26\n    at Request._callback (/root/github.com/AschPlatform/asch/src/core/transport.js:731:11)\n    at Request.self.callback (/root/github.com/AschPlatform/asch/node_modules/request/request.js:187:22)\n    at emitTwo (events.js:126:13)\n    at Request.emit (events.js:214:7)\n    at Request.<anonymous> (/root/github.com/AschPlatform/asch/node_modules/request/request.js:1044:10)\n    at emitOne (events.js:116:13)\n    at Request.emit (events.js:211:7)' }
info 2018-02-06 12:49:50 413 app.js:192 Cleaning up... 
info 2018-02-06 12:49:50 414 dapps.js:2403 all dapps stopped successfully 
info 2018-02-06 12:49:50 414 app.js:203 Cleaned up successfully 
info 2018-02-06 12:49:50 415 app.js:218 process exited 
```

**Pull request merged**